src/test/integration/users/User.t.sol

```diff
@@ -56,20 +25,12 @@ contract User is Logger, IDelegationManagerTypes, IAllocationManagerTypes {
     EigenPod public pod;
     uint40[] validators;
 
-    constructor(string memory name) {
-        IUserDeployer deployer = IUserDeployer(msg.sender);
+    ConfigGetters public config;
 
-        // TODO uncommented for reason above
-        // allocationManager = deployer.allocationManager();
-        // permissionController = deployer.permissionController();
-        delegationManager = deployer.delegationManager();
-        strategyManager = deployer.strategyManager();
-        eigenPodManager = deployer.eigenPodManager();
-
-        timeMachine = deployer.timeMachine();
-        beaconChain = deployer.beaconChain();
-
-        _createPod();
+    constructor(string memory name) {
+        config = ConfigGetters(address(msg.sender));
```

Comment:
Why not just inherit ConfigGetters?

I'm actually a little lost on what's going on here - are we calling methods on User when we access `config.delegationManager()`?